packages/terra-time-input/src/TimeInput.module.scss

```
@@ -11,11 +11,11 @@
   }
 
   .time-input {
-    background-color: var(--terra-time-input-background, #fff);
-    background-image: var(--terra-time-input-animation-background, none);
+    background-color: var(--terra-time-input-background-color, #fff);
+    background-image: var(--terra-time-input-animation-background-image, none);
```

Drop the word animation
```suggestion
    background-image: var(--terra-time-input-background-image, none);
```